lib/privatekey.js

```diff
@@ -157,14 +157,18 @@ PrivateKey._transformBuffer = function(buf, network) {
   }
 
   info.network = Networks.get(buf[0], 'privatekey');
```

Comment:
This does not make a sense, `info.network` redefined later. Probably someone wanted make fast check based on `Networks.get`, but it wasn't done in old code.